cmd/cluster-agent/api/v1/languagedetection/language_detection.go

```diff
@@ -51,8 +51,8 @@ var ownersLanguagesOnce sync.Once
 func loadOwnersLanguages(wlm workloadmeta.Component) *OwnersLanguages {
 	ownersLanguagesOnce.Do(func() {
 		ownersLanguages = *newOwnersLanguages()
-		languageTTL = config.Datadog.GetDuration("language_detection.cleanup.language_ttl")
-		cleanupPeriod := config.Datadog.GetDuration("language_detection.cleanup.period")
+		languageTTL = config.Datadog.GetDuration("cluster_agent.language_detection.cleanup.language_ttl")
+		cleanupPeriod := config.Datadog.GetDuration("cluster_agent.language_detection.cleanup.period")
```

Comment:
Could we pass the config.Component to `cmd/cluster-agent/api/`.`StartServer` function in https://github.com/DataDog/datadog-agent/blob/dacd99ae16b6e7017add79937ca4b64b663c30cd/cmd/cluster-agent/subcommands/start/command.go#L256 and use the component to get the config values?

Reply:
Based on our previous discussion, leaving this for a separate PR.